Engineering/Software-Engineering/L4-Lead-Software-Engineer.md

```diff
@@ -11,84 +11,85 @@
+- I confidently tackle uncertain or risky situations with minimal upfront information and oversight.
+- I make thoughtful architectural decisions, addressing cross-cutting concerns like infrastructure, security, scalability, and maintainability.
+- I stay informed on evolving standards, platform features, and key vendor and community technologies.
 
 <details>
 <summary>Examples</summary>
 
 - I successfully completed a large, complex project with multiple components.
-- I dived head-first into a bug fix or incident in an area of significant ambiguity or risk, led the investigation and resolved the issue.
-- I engaged with experts on another team for help approaching a complex problem outside of my domain and then ensured that my team understood the response and were able to implement the advice consistently in our work.
+- I led the investigation and resolution of a bug or incident in a highly ambiguous or risky area.
+- I consulted with experts on a complex problem outside my domain and ensured their insights were applied effectively by my team.
 
 </details>
 
 ## 🌱 Teaching and Mentoring
 
-- I define, document, and evolve best practices for development in my team's area of focus.
-- I shepherd and aid in development of new projects across the organization.
-- I am constantly working to broaden the technical capabilities of my team.
-- I am comfortable with challenging and coaching my peers.
+- I define, document, and evolve best practices in my team's area of focus.
+- I help guide and support the development of new projects across the organisation.
+- I work to expand the technical skill sets of my team.
+- I challenge and coach my peers to improve and grow.
 - I help people determine the most valuable ways they can contribute, and grow, and I help them block out the noise.
+- I help teammates prioritise their efforts, develop their strengths, and stay focused on impactful work.
 
 <details>
 <summary>Examples</summary>
 
-- I provided training and mentoring for multiple team members, deliberately helping them to round out their skill sets.
-- I shared my on-the-job learning and experiences with others so they can understand and be more effective in their own roles.
-- I broke up a project in such a way that lined up appropriate challenges for each of my teammates that helped them grow.
-- I had some difficult conversations with my teammates, challenging them directly while showing them my care for them personally.
-- I adjusted my interactions to adapt to the different working styles and needs of individuals on my team.
+- I trained and mentored team members, helping them develop well-rounded skill sets.
+- I shared my experiences to help others become more effective in their roles.
+- I divided a project into tasks that presented appropriate challenges to help teammates grow.
+- I had constructive conversations that provided direct feedback while showing personal care for teammates.
```

Comment:
I feel like "constructive" is a euphemism here which actually masks the intent of the statement. While ideally these conversations should be constructive, I think the fact that they are _difficult_ (i.e. not just providing someone some uncontroversial but constructive feedback) is the mark of an L4. I'd hope an L3 could provide radically candid feedback of an uncontroversial nature.

Reply:
Good signal, thank you. Reverted to original.